src/test/java/org/apache/commons/lang3/builder/ReflectionDiffBuilderTest.java

```diff
@@ -29,6 +29,31 @@ private static final class TypeTestChildClass extends TypeTestClass {
         String field = "a";
     }
 
+    @SuppressWarnings("unused")
+    private static class TypeTestEntityClass implements Diffable<TypeTestEntityClass> {
```

Comment:
Rename `TypeTestEntityClass` to `EntityTestClass`.
Add a class Javadoc that states the intent for the test fixture to emulate a JPA entity class.